src/server/utils/my-repos.ts

```diff
@@ -25,66 +26,89 @@ const MAX_STATS_RETRIES = 5;
 // Variables that impact the scoring algorithm.
 
 // Repos below this score are discarded from the list.
-const SCORE_THRESHOLD = 20;
+const SCORE_THRESHOLD = 30;
 // A contribution is roughly a commit. This parameter impacts the distribution &
 // scale of the resulting scores.
 const CONTRIBUTION_SIZE = 10;
-const CONTRIBUTION_WINDOW = 1000 * 60 * 60 * 24 * 90;  // 90 days.
+const CONTRIBUTION_WINDOW = 1000 * 60 * 60 * 24 * 7;  // 7 days.
 
 /**
  * Returns a list of repos that the given user contributes to. It is ordered
  * based on contribution recency and consistency.
  */
 export async function getMyRepos(
```

Comment:
I'd rename this to `getAutomatedRepoList`, just to differentiate between the stored repo's vs the first load of repos.

Reply:
Done.